apps/relgen/src/index.ts

```diff
@@ -376,11 +376,11 @@
     let prompt: string | undefined;
 
     if (templateFile) {
-      template = await readFile(templateFile, 'utf-8');
+      template = (await readFile(templateFile, 'utf-8')).trim();
     }
 
     if (promptFile) {
-      prompt = await readFile(promptFile, 'utf-8');
+      prompt = (await readFile(promptFile, 'utf-8')).trim();
```

Comment:
The use of `trim` is not allowed as per the rules. Please find an alternative approach to handle any unwanted whitespace.